charts/console-api/templates/doppler-secret.yaml

```
@@ -0,0 +1,17 @@
+apiVersion: secrets.doppler.com/v1alpha1
+kind: DopplerSecret
+metadata:
+  name: {{ include "console-api.fullname" . }}-dopplersecret
+  namespace: {{ .Values.doppler.namespace | default "doppler-operator-system" }}
```

afaik this is static. I don't mean this needs to change, it would work anyways

```suggestion
  namespace: doppler-operator-system
```